### PR TITLE
docs: improve accessibility of tohpt3

### PR DESCRIPTION
**Pull request by kapunahelewong**
Fixes contrast issues.
Makes labels explicitly reference inputs.
Removes the term master.
Removes/replaces "simple" wording.
Adds padding to input so it is an easier target for touch or pointer.
Changes app to application.
Changes "name: " label to "Hero name: ".

### CODE REVIEW

File: aio/content/tutorial/toh-pt3.md [hunk 1]
```diff
@@ -79,13 +79,12 @@ Add a `hero` property, preceded by the `@Input()` decorator.
...
 There are no more properties. There's no presentation logic.
-This component simply receives a hero object through its `hero` property and displays it.
+This component only receives a hero object through its `hero` property and displays it.
 
 ## Show the `HeroDetailComponent`
 
-The `HeroesComponent` is still a master/detail view.
-
-It used to display the hero details on its own, before you cut that portion of the template. Now it will delegate to the `HeroDetailComponent`.
+The `HeroesComponent` used to display the hero details on its own, before you cut that portion of the template.
```
Comment (petebacondarwin) [inline aio/content/tutorial/toh-pt3.md:86]:
```suggestion
The `HeroesComponent` used to display the hero details on its own, before you removed that portion of the template.
```
I feel that `removed` is less colloquial than `cut`.

**Review by petebacondarwin — APPROVED** [review]
One NIT.

**Review by jelbourn — APPROVED** [review]
LGTM aside from Pete's comments

File: aio/content/examples/toh-pt3/src/app/hero-detail/hero-detail.component.html [hunk 2]
```diff
@@ -3,9 +3,8 @@
   <h2>{{hero.name | uppercase}} Details</h2>
   <div><span>id: </span>{{hero.id}}</div>
   <div>
-    <label>name:
-      <input [(ngModel)]="hero.name" placeholder="name"/>
-    </label>
+    <label for="hero-name">Hero name: </label>
+    <input id="hero-name" [(ngModel)]="hero.name" placeholder="name"/>
```
Comment (IgorMinar) [inline aio/content/examples/toh-pt3/src/app/hero-detail/hero-detail.component.html:7]:
```suggestion
    <input id="hero-name" [(ngModel)]="hero.name" placeholder="name">
```

Comment (IgorMinar) [reply aio/content/examples/toh-pt3/src/app/hero-detail/hero-detail.component.html:7]: the ending `/` is unnecessary - it's needed only when writing xhtml which is not the case here.

**Review by IgorMinar — APPROVED** [review]
one small nit, but overall lgtm. thanks!

Reviewed-for: docs-getting-started-and-tutorial

File: aio/content/examples/toh-pt3/src/app/hero-detail/hero-detail.component.html (hunk 2 above)
Comment (gkalpak) [reply aio/content/examples/toh-pt3/src/app/hero-detail/hero-detail.component.html:7]: &emsp;&ensp;<sub><sub>😞</sub></sub>
&emsp;<sub>😞</sub>
&nbsp;&nbsp;:disappointed:
&nbsp;<sup>&nbsp;😞</sup>
<sup><sup>&ensp;😞</sup></sup>

Comment (gkalpak) [reply aio/content/examples/toh-pt3/src/app/hero-detail/hero-detail.component.html:7]: Quotes around attribute values and most of the whitespace are also not needed 😛
Just sayin' :trollface:

Comment (petebacondarwin) [reply aio/content/examples/toh-pt3/src/app/hero-detail/hero-detail.component.html:7]: To be fair, @gkalpak, it is now standard convention not to use a closing slash in void elements in HTML and  can always be done. Whereas removing whitespace and quotations is context dependent. You can't always remove quotes if the quoted value contains whitespace for instance. So the conventions are not directly comparable.

Comment (gkalpak) [reply aio/content/examples/toh-pt3/src/app/hero-detail/hero-detail.component.html:7]: Fair enough 😃

**Review by aikithoughts — APPROVED** [review]
LGTM!
Reviewed-for: global-docs-approvers

**Pull request closed without merging**